### DISCUSSION TITLE
How to activate breadcrumbs?

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5184] [General]
Hello, I have a question about breadcrumbs.
Into the platform configuration page, I found the "navigation bar in course home page" setting and set it to "initial page" but nothing happens. I also tried to uncomment this line

`$_configuration['breadcrumb_hide'] = false;`

but again nothing happens so I'm assuming that I'm doing something wrong since I was expecting to see something like
"Home > Category > Course Name" or
"Home > Course Name"

So the question is, is there a way to activate breadcrumbs the way I expected or am I misinterpreting the purpose of these settings?

Thanks is advance

**Comment by ywarnier** [comment]
Hi @a-mian-lateral 
Your post is a bit imprecise. There are 2 settings you seem to be referring to.

One is `Display course navigation menu` which will show additional icons representing the other course tools when you are inside of a course tool (avoiding you to return to the course homepage to change from one tool to another).

The other is `Course homepage breadcrumb` which only changes the information shown in the breadcrumb *when you are on the course homepage*, but the only options are 4 things that do *not* include the category.

Then, about `$_configuration['breadcrumb_hide'] = false;`, this setting has to be uncommented *and* set to true to take effect, but it will effectively *hide* the breadcrumb, which is not really what you're looking for.

We don't show the course category by default because there's not always a course category and we like consistency, but you could change main/inc/lib/banner.lib.php, in function `return_breadcrumb()`, to add the category (which you should find in `$courseInfo['categoryName']` right there).
